setup.py

```diff
@@ -6,13 +6,14 @@
 
 setup(
     name="Open edX",
-    version="0.4",
+    version="0.5",
```

Comment:
In ansible, we already install the local.txt requirements file on each build.  This shouldn't be an issue in production.  If paver does not do this, it's a bug and should be fixed by the development team.